bokehjs/src/lib/embed/server.ts

```diff
@@ -21,6 +22,11 @@ export function _get_ws_url(app_path: string | undefined, absolute_url: string |
     loc = window.location
   }
 
+  let protocol = "ws:"
+  if (loc.protocol == "https:") {
+    protocol = "wss:"
+  }
+
```

Comment:
Also let's get rid of a mutable variable:
```suggestion
  const protocol = loc.protocol == "https:" ? "wss:" : "ws:"
```